virttest/qemu_vm.py

```diff
@@ -384,6 +384,17 @@ def _add_option(option, value, option_type=None, first=False):
                 return fmt % (option, str(value))
             return ""
 
+        def _get_pci_bus(params, name=None):
+            """
+            Get device pattern pci bus.
+
+            :param params: device params
+            :param name: deivce type name
+            """
+            name = name and "%s_pci_bus" % name or "pci_bus"
+            pci_bus = {"aobject": params.get(name, "pci.0")}
+            return pci_bus
```

Comment:
Oh actually there is a minor bug in this function. The default should be:

```
pci_bus = {"aobject": object_params.get(name, params.get("pci_type", "pci.0"))}
```
so it reflects the main chosen `pci_type` type.